external-crates/move/crates/move-compiler/tests/move_2024/ide_mode/use_incomplete.move

```diff
@@ -0,0 +1,49 @@
+module a::m {
```

Comment:
This should also have `use_incomplete.ide` and `use_incomplete.ide.exp`

Reply:
Excluding these was a conscious decision actually - because the `use` auto-completion for "regular" uses (and these are the only ones in this file) does not use `AliasAutocompleteInfo` at all, it felt like adding these would only generate unnecessary noise. I can certainly add them of course! Also, perhaps we should keep this test outside of `ide_mode` directory, but I still thought it's the best location for it

Reply:
There are ways to avoid that, but it seems fine to just land without it since we're doing this for a different purpose.